book/push/push-ios.md

```diff
@@ -0,0 +1,90 @@
+
+# Push plugin - iOS
+
+The iOS push plugin for Zapp is based on implementing the `ZPPushProviderProtocol`.
+The protocol goes through all of the functions for initializing a push provider.
+
+## Create a new push provider
+
+This chapter describes how to build a push provider plugin. What kind of preparations are necessary, which protocol methods should be implemented etc.
+
+### General Implementation
+
+Before you start, please do the following important steps:
+
+1. In your cocoapods files, add a dependency to `ZappPushPluginsSDK`. This dependency is a mandatory dependency for your push provider plugin. The dependency is available from the public repository cocoapods source `git@github.com:applicaster/PluginsBuilderCocoaPods.git`.
+2. Create a new class for you plugin adapter that inherits from ZPPushProvider.
+3. Import `ZappPushPluginsSDK` and `ZappPlugins` 
+4. Start the plugin development
```

Comment:
either have this in both guides - or better - remove it from here

Reply:
Added it also to android